README.md

```diff
@@ -9,7 +9,7 @@ designed with unique features that help with more specific use cases.
 
   - [General model](#general-model)
   - [Layer configuration examples](#layer-configuration-examples)
-  - [Running pebble](#running-pebble)
+  - [Using pebble](#using-pebble)
   - [Layer specification](#layer-specification)
```

Comment:
In the Table of Contents / the organization of the sections, please consider using Diataxis. That is, you should the 5 main sections below, with your existing sections nested within along the following lines (at least, at first glance):

- Home
- Tutorial
- How-to guides
    - Install (i.e., How to install Pebble)
    - Set up (i.e., How to set up the Pebble directory)
    - Use (i.e., How to use Pebble)
    - Run the daemon (i.e., How to run the Pebble daemon)
    - Start and stop services (i.e., How to start and stop Pebble services)
- Reference
    - General model
    - Layer configuration spec + examples
    - `pebble` CLI
        - List of `pebble` CLI commands
    - Service dependencies
    - Health checks
    - Changes and tasks
    - ...
- Explanation

At first glance, all the materials you have are either How-to guides or Reference, so you can leave out the Tutorial and Explanation for now. As for how to organize How-to guides vs. Reference: I'd put any material that defines things into Reference and any material that gives instructions into How-to guides. I'd also try to organize the Reference docs around primitive concepts and bits and then the How-to guides also around that where, for a product that has a CLI, the how-to guides are usually about manipulating a primitive entity in various ways using various CLI commands. See, for example, https://juju.is/docs/olm/the-juju-client and https://juju.is/docs/olm/manage-juju, or https://juju.is/docs/olm/network-spaces and https://juju.is/docs/olm/manage-spaces , or https://juju.is/docs/olm/action and https://juju.is/docs/olm/manage-actions .

Reply:
This is a good discussion, however, it's a big structural change to the rest of the document -- I'm going to leave this for future work.